### ISSUE TITLE
Support for Sonos Arc soundbar?

### ISSUE THREAD

**Issue by Maykel1969** [ebaauw/homebridge-zp#159]
Hi,
I try to get my Sonos Arc running with this plugin in Hoobs.
I know that this speaker is not (yet) supported but I hope it’s coming.
In homekit I only get an on/off switch but the log shows much more info.
For example if I cange the volume with the sonos app then I see his in the log.
But why do I just get an on/off switch? Do I need to be more patient and wait for support?
If you need a tester, just let me know.
Best regards from the netherlands.

**Comment by ebaauw** [comment]
The Arc should be fully supported.

Please see the README:
> Please note that Siri and even Apple's Home app still provide only limited HomeKit support. To use the full features of Homebridge Zp, you might want to check out some other HomeKit apps, like the Eve app (free) or Matthias Hochgatterer's Home+ app (paid).

**Comment by Maykel1969** [comment]
You are right, the eve app shows much more information. However I like the home app much more because of its simplicity. Also the WAF (Wife Acceptance Factor) of the Eve app is much lower compare to the simple home app.
I’ll have to live with the shortcomings I guess.
Thanks for your quick feedback!

**Comment by Zer0x00** [comment]
You can set ```service``` to fan, this will allow you to adjust the volume of your Arc while using the Home app.

**Comment by Maykel1969** [comment]
That’s a nice option! It gives me a but more control!
Thanks!
